cmd/plural/cd_settings.go

```diff
@@ -0,0 +1,53 @@
+package plural
+
+import (
+	"os"
+
+	consoleclient "github.com/pluralsh/console-client-go"
+	"github.com/pluralsh/plural-cli/pkg/utils"
+	"github.com/urfave/cli"
+	"sigs.k8s.io/yaml"
+)
+
+func (p *Plural) cdSettings() cli.Command {
+	return cli.Command{
+		Name:        "settings",
+		Subcommands: p.cdSettingsCommands(),
+		Usage:       "manage CD settings",
+	}
+}
+
+func (p *Plural) cdSettingsCommands() []cli.Command {
+	return []cli.Command{
+		{
+			Name:      "agents",
+			ArgsUsage: "FILENAME",
+			Action:    latestVersion(requireArgs(p.handleUpdateAgents, []string{"FILENAME"})),
+			Usage:     "update agents settings",
+		},
+	}
+}
+
+func (p *Plural) handleUpdateAgents(c *cli.Context) error {
```

Comment:
I think this subcommand would ideally only set the `agentValues` field of the global settings, and we can add other subcommands for other fields (it's a bit clunky to update everything at once in the cli)

Reply:
done